api/api.py

```diff
@@ -35,7 +35,7 @@
     'uid': '[0-9a-zA-Z.@_-]*',
 
     # Container name
-    'cname': 'projects|sessions|acquisitions|collections',
+    'cname': 'projects|sessions|acquisitions|collections|analyses',
```

Comment:
I think this now also allows for the removal of [250-251](https://github.com/scitran/core/pull/836/files#diff-e0978ea022e5e946d30dfa3911881b5eR250) and [260-267](https://github.com/scitran/core/pull/836/files#diff-e0978ea022e5e946d30dfa3911881b5eR260), cool! (Also refererhandler.py#116-)